net/openconnect/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 
 name                openconnect
-version             8.03
+version             8.05
 revision            1
```

Comment:
The `revision` needs to be set back to 0 when increasing the `version`.

Reply:
Thank you, resolved by 2993b84